utils/check_copies.py

```diff
@@ -492,6 +492,7 @@ def check_model_list_copy(overwrite=False, max_per_line=119):
     "Speech2Text2",
     "Vision Encoder decoder",
     "VisionTextDualEncoder",
+    "GPT NeoX Japanese",
```

Comment:
I think the model should be in the README and this line removed.

Reply:
I removed here.